h/groups/schemas.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+import colander
+
+from h.accounts.schemas import CSRFSchema
+
+
+class GroupSchema(CSRFSchema):
+    name = colander.SchemaNode(colander.String(),
+                               validator=colander.Length(min=4, max=100))
```

Comment:
Minimum is so we have *something* reasonable for a slug. Maximum: see the chinchillas.